flytekit/core/data_persistence.py

```diff
@@ -219,32 +234,38 @@ def put(self, from_path: str, to_path: str, recursive: bool = False, **kwargs):
             from_path, to_path = self.recursive_paths(from_path, to_path)
         return file_system.put(from_path, to_path, recursive=recursive, **kwargs)
 
-    def get_random_remote_path(self, file_path_or_file_name: typing.Optional[str] = None) -> str:
```

Comment:
Should we deprecate it first? Someone might use it in their code.

Reply:
yeah i can add it back... in a revised form.